stl/inc/limits

```diff
@@ -1116,6 +1116,19 @@ _NODISCARD int _Checked_x86_x64_popcount(const _Ty _Val) noexcept {
 }
 #endif // _HAS_POPCNT_INTRINSICS
 
+#if defined(_M_ARM64EC) && !defined(_M_CEE_PURE) && !defined(__CUDACC__) && !defined(__INTEL_COMPILER)

```

Comment:
Why only `_M_ARM64EC` and not usual `_M_ARM64` ?

Reply:
`intrin0.h` uses `_M_ARM64EC`
```
#pragma once
#define __INTRIN0_H_

#ifdef _M_ARM64EC
#include <intrin.h>
#else  // _M_ARM64EC
#include <intrin0.inl.h>
#endif // _M_ARM64EC
```
and 
https://github.com/microsoft/STL/blob/a9321cfe53ea31a7e197d5d8336167d6ca3de8b6/stl/inc/limits#L1087-L1092

These are my reasons for using `_M_ARM64EC`
But I don't know maybe we need use `_M_ARM64`...

Reply:
I'm pretty sure we need `_M_ARM64`. And we need to make popcount part available from `<intrin0.inl.h>`.
Since `<intrin0.h>` and  `<intrin0.inl.h>` are not under your control, you should conditionally replace `#include <intrin0.h>` with `#include <intrin.h>`, the condition is _M_ARM64 && !_M_ARM64EC, and mark it with `TRANSITION, GH-XXXX` comment, where `GH-XXXX` is an issue where you ask to add the stuff you need into `<intrin0.h>`. Yes, it is that hard.

Reply:
Think the EC flavor is not even tested on GitHub, so don't be impressed if test fails after you fix this

Reply:
Actually, I only now read what is `_M_ARM64EC` :)
Yes, 100%, we need `_M_ARM64` 

https://twitter.com/never_released/status/1262132009574502403